internal/jstest/jstest.go

```diff
@@ -85,8 +85,14 @@ func Run(ctx context.Context, dir string, args []string) (*internal.TestResults,
 		out  []byte
 	}
 
-	// tokens is a counting semaphore used to enforce a limit of 20 concurrent calls to runShellWithScript.
```

Comment:
What is the new behaviour now? A comment would be useful here :hugs: 

If I understand correctly, on CI `FILE_CONCURRENCY` is set to 1, when is `token = 50` being used?

Reply:
`$GITHUB_ENV` is just for CI and will not effect local runs. The above behavior is a slight refactor and an increase in the number of concurrent files being processed. This was to have tests complete faster locally.

Reply:
Just a suggestion. I think we may want to improve naming of `FILE_CONCURRENCY` -> `tokens` -> `sema`. It's easy to get confused what is `FILE_CONCURRENCY` is :thinking:

Reply:
I changed it to `NUM_TOKENS`, not sure what else to call it in this context.

Reply:
Changed to `WORKERS` as the name implies its purpose.